source/release-notes/1.7.txt

```diff
@@ -0,0 +1,31 @@
+.. _c2c-release-notes-1.7:
+
+===============================
+Release Notes for mongosync 1.7
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.7.0-c2c-release-notes:
+
+1.7.0 Release
+-------------
+
+**Upcoming**
+
+Time Series Collections
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in ``mongosync`` 1.7.0, time series collections are
+synchronized between clusters.
+
+Minimum Supported Version
+-------------------------
+
+In 1.7.0, the minimum supported versions of MongoDB are 6.0.8 and 7.0.0.
```

Comment:
ah so we'd want to include these minimum supported versions for the 1.6 release notes. We should hold off on listing this for 1.7 until we start its release process

Reply:
@ajayvijayakumar123 Will do. Also discussed with Martin and will only list 6.0.8, and hold off merging 1.7 tickets until Martin lets me know it is good for release. In the meantime, is the rest of the content good? Cheers.